Sources/Testing/Issues/Issue.swift

```diff
@@ -300,6 +318,8 @@ extension Issue.Kind {
           .expectationFailed(Expectation.Snapshot(snapshotting: expectation))
       case let .confirmationMiscounted(actual: actual, expected: expected):
           .confirmationMiscounted(actual: actual, expected: expected)
+      case let .confirmationOutOfRange(actual: actual, expected: _):
+          .confirmationMiscounted(actual: actual, expected: 0)
```

Comment:
Why not also adding a corresponding new case for Snapshot here?

Is this on purpose or this is a todo item?

Reply:
We are not planning to continue developing the snapshot types. See #479.